HHS/Head-Start-TTADP
# [TTAHUB-1530] Make certain goal templates selectable from the AR

## Description of change
This PR inserts via migration a goal template that will be selectable from the user interface. When selected, it will trigger another question being added to the goal form. That question has dynamic validation and text, also inserted via migration.

## How to test
- Select the goal "(FEI) The recipient will eliminate and/or reduce underenrollment as part of the Full Enrollment Initiative (as measured by monthly reported enrollment)" on a new activity report
- It should be available under any recipient.
- When you select that goal, confirm that a conditional question appears and that you can only select a maximum two answers to that question.
- Save the report, and return, and confirm that the goal appears as expected
- Approve the report and visit the approved report page and the RTR and confirm that that goal a) appears correctly on the approved report and b) appears on the RTR for that recipient


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1530


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
